api/openapi-spec/swagger.json

```diff
@@ -15837,6 +15837,15 @@
      }
     }
    },
+   "v1.FeatureIOAPIC": {
+    "type": "object",
+    "properties": {
+     "driver": {
+      "description": "Configure the guest I/O APIO mode('kvm' or 'qemu').",
```

Comment:
Besides QEMU, we also support Rust based VMM, e.g., Cloud hypervisor

Reply:
Currently, the patch only consider using libvirt to start the VM.  More works were needed to cover the Cloud hypervisor support.

Reply:
> Currently, the patch only consider using libvirt to start the VM. More works were needed to cover the Cloud hypervisor support.

Got it. We should consider RUST based VMM later.